brooklyn-tosca-transformer/pom.xml

```diff
@@ -154,8 +154,8 @@
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
-            <version>1.9.0</version>
+            <artifactId>mockito-core</artifactId>
+            <version>2.7.12</version>
```

Comment:
I would prefer this to be inherited from the parent pom as ie `<version>${mockito-core.version}</version>`